docs/platforms/javascript/guides/connect/index.mdx

```diff
@@ -30,34 +30,38 @@ import * as Sentry from "@sentry/node";
 // const connect = require('connect');
 // const Sentry = require('@sentry/node');
 
-// Must configure Sentry before doing anything else with it
-Sentry.init({ dsn: "___PUBLIC_DSN___" });
+const app = connect();
 
-function mainHandler(req, res) {
-  throw new Error("My first Sentry error!");
-}
+// Sentry must be initialized before doing anything else with it
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
 
-function onError(err, req, res, next) {
-  // The error id is attached to `res.sentry` to be returned
-  // and optionally displayed to the user for support.
-  res.statusCode = 500;
-  res.end(res.sentry + "\n");
-}
+  // Adding Performance Monitoring by setting tracesSampleRate
+  // Set tracesSampleRate to 1.0 to capture 100% of transactions
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0,
+});
+
+// The request handler must be the first middleware on the app
+app.use(Sentry.Handlers.requestHandler());
+// Performance Monitoring: creating a trace for every incoming request
+app.use(Sentry.Handlers.tracingHandler());
 
-connect(
```

Comment:
is this example here just wrong, or old, or a different way to do the same thing, I wonder? 🤔

Reply:
It's wrong. `connect` does not accept a parameter and `connect.bodyParser()` and `connect.cookieParser()` is also not possible as those function are not available.

Reply:
LOL, ok, great find xD